network-api/networkapi/buyersguide/templates/about/contest.html

```diff
@@ -0,0 +1,377 @@
+{% extends "../bg_about_without_nav.html" %}
+
+{% comment %}
+  This is legal text and should only be translated by qualified lawyers.
+  For this purpose we're not translating this template.
+{% endcomment %}
```

Comment:
Copied the contest.html page to the namespaced dir. Only thing I added was this comment.